cumulusci.yml

```diff
@@ -62,9 +66,26 @@ flows:
                 task: assign_pset
                 ignore_failure: True
 
+    install_regression:
+        steps:
+            5:
+                task: deploy_delete_config
+            6:
+                task: deploy_dev_config
+                options:
+                    unmanaged: False
+                    namespace_inject: $project_config.project__package__namespace
+            7:
+                task: deploy_qa_config
+            8:
+                task: assign_pset
```

Comment:
@davidmreed install_prod, install_beta, and install_regression all use the `config_managed` flow. I think we want to add these steps there rather than only for install_regression, since we want them for all managed installations.

Reply:
@davisagli I wasn't clear that we did want them for all managed installations. The Page Layout, in particular, is an odd artifact because it's V4S only (no NPSP fields), and meant specifically for regression testing. @kselvocki do you any input here?

Reply:
@davidmreed hmm, fair question. if it's only for QA purposes it should probably go in a separate metadata bundle that is deployed for qa_org and install_regression but not for install_prod and install_beta (like how in NPSP we have separate `trial` and `qa` metadata bundles)

Reply:
I've tweaked how the bundles get installed to ensure that `qa_org` and `install_regression` both include the same metadata.